main.go

```diff
@@ -114,6 +114,7 @@ func mainWithError() error {
 	daemonCommand.PersistentFlags().String(f.Service.CNR.Address, "https://quay.io", "Address used to connect to CNR, defaults to quay's managed offering.")
```

Comment:
@rossf7 is this duplicate now i.e. can it be removed?

Reply:
It's used for pulling tarballs from Appr. We need both because in China we pull images from Aliyun but tarballs are still from Quay.